css/styles.css

```diff
@@ -0,0 +1,97 @@
+* {
+  font-family: sans-serif;
+}
+
+body {
+  background-color: #edebed;
+}
+
+header {
+  background-color: #4a2373;
+}
+
+h1.journal-header {
+  color: white;
+  font-weight: 400;
+  font-size: 1rem;
+}
+
+h2.page-title {
+  font-size: 1.5rem;
+}
+
+img.entry-image {
+  width: 100%;
+  height: 15rem;
```

Comment:
We don't want to give this a `width` & a `height`. We only want to give this a `width`. We want the browser to automatically adjust the `height` for us, so it'll maintain the aspect ratio. By setting a `height`, you're distorting the image.

Reply:
Hi Shawn, thank you for your comment. I address this issue by removing the `height` property. However, as I saw that in the demo of issue #1, the image became a rectangle in mobile view and a square in desktop view with the center of the image undistorted. Therefore, in mobile view, I set a height and set object-fit: cover so the image is cropped to fit the view, but it stays undistorted.